types/office-js/index.d.ts

```diff
@@ -28,126 +37,243 @@ declare namespace Office {
          */
         iOS,
         /**
-         * Platform is Android
+         * The platform is an Android device.
          */
         Android,
         /**
-         * Platform is Winrt
+         * The platform is WinRT.
          */
         Universal
     }
     // Objects
-    export interface AsyncResult {
+        /**
+		* An object which encapsulates the result of an asynchronous request, including status and error information if the request failed.
+		*         
+        * @remarks
+        * Hosts: Access, Excel, Outlook, PowerPoint, Project, Word
+        *
+        * Last changed in: 1.1
+        * 
+        * When the function you pass to the `callback` parameter of an "Async" method executes, it receives an AsyncResult object that you can access from the `callback` function's only parameter.
+		*/
+        export interface AsyncResult {
+        /**
+        * Gets the user-defined item passed to the optional `asyncContext` parameter of the invoked method in the same state as it was passed in. This the user-defined item (which can be of any JavaScript type: String, Number, Boolean, Object, Array, Null, or Undefined) passed to the optional `asyncContext` parameter of the invoked method. Returns Undefined, if you didn't pass anything to the asyncContext parameter.
+        *         
+        * @remarks
+        * Hosts: Access, Excel, Outlook, PowerPoint, Project, Word
+        * 
+        * Last changed in: 1.1
+        */
         asyncContext: any;
+        /**
+        * Gets the [AsyncResultStatus](office.asyncresultstatus.md) of the asynchronous operation.
+        * 
+        * @remarks
+        * Hosts: Access, Excel, Outlook, PowerPoint, Project, Word
+        * 
+        * Last changed in: 1.1
+        */
         status: AsyncResultStatus;
+        /**
+        * Gets an [Error](office.error.md) object that provides a description of the error, if any error occurred.
+        * 
+        * @remarks
+        * Hosts: Access, Excel, Outlook, PowerPoint, Project, Word
+        * 
+        * Last changed in: 1.1
+        */
         error: Error;
+        /**
+        * Gets the payload or content of this asynchronous operation, if any.
+        
+        * @remarks
+        * You access the AsyncResult object in the function passed as the argument to the callback parameter of an "Async" method, such as the `getSelectedDataAsync` and `setSelectedDataAsync` methods of the Document object.
+        * 
+        * Note: What the value property returns for a particular "Async" method varies depending on the purpose and context of that method. To determine what is returned by the value property for an "Async" method, refer to the "Callback value" section of the method's topic. For a complete listing of the "Async" methods, see the Remarks section of the AsyncResult object topic.
+        * 
+        * Hosts: Access, Excel, Outlook, PowerPoint, Project, Word
+        * 
+        * Last changed in: 1.1
```

Comment:
Here and elsewhere:  what does "Last changed in 1.1" mean.
Both the last-changed part (why do we care when it was last changed, esp. when with most APIs it's "when was it introduced").  And what does 1.1 mean (Office.js 1.1?  But pretty much the whole thing is 1.1, Office.js 1.0 doesn't really exist anymore for 99.9% of developers).
API sets would be meaningful for most APIs, though AsyncResult is the one exception (it's a return type, not an API directly), so I would just omit it.

Reply:
Ditto for everywhere else where you have "Last changed in..."